server/src/main/java/org/elasticsearch/search/internal/SearchContext.java

```diff
@@ -328,11 +308,9 @@ public final void assignRescoreDocIds(RescoreDocIds rescoreDocIds) {
 
     public abstract int[] docIdsToLoad();
 
-    public abstract int docIdsToLoadFrom();
-
     public abstract int docIdsToLoadSize();
 
-    public abstract SearchContext docIdsToLoad(int[] docIdsToLoad, int docsIdsToLoadFrom, int docsIdsToLoadSize);
+    public abstract SearchContext docIdsToLoad(int[] docIdsToLoad, int docsIdsToLoadSize);
```

Comment:
We always seem to call this with `(array, array.length)` so maybe we can remove the second parameter here as well?

Reply:
I found one instance where this isn't the case in `SearchService#executeFetchPhase`, where these parameters are taken from `IntArrayList`. This could likely be refactored, but we can save it for another time!